### DISCUSSION TITLE
Yellow dots near Jupiter but not other planets or objects

### DISCUSSION THREAD

**Discussion by andrewbernard** [Stellarium/stellarium#3769] [Q&A] [answered]
What do these dots represent and why do they only show for Jupiter? Version 24..1

![image](https://github.com/Stellarium/stellarium/assets/4009723/b8ce6cb8-d971-4640-a46c-c887640dc13d)

**Comment by 10110111** [comment, marked as answer]
These look like markers from ephemeris computations. Have you been doing any astronomical calculations in Stellarium?

**Reply by andrewbernard** [reply]
Indeed yes I have. I'm a newbie to Srellarium. What a totally amazing program.

**Reply by 10110111** [reply]
So you can remove these markers if you press _Cleanup ephemerides_ in the AstroCalc dialog.
